test/scripts/dag_fact.js

```diff
@@ -0,0 +1,58 @@
+const Helper = require('hubot-test-helper');
+const sinon = require('sinon');
+const facts = require('../../scripts/dag_fact.coffee');
+
+const helper = new Helper('../../scripts/dag_fact.coffee');
+
+const co = require('co');
+const expect = require('chai').expect;
+
+describe('daggity facts', () => {
+  // Replace the list of facts with one known fact, so we don't
+  // have to worry about randomness.
+  facts.factList.splice(0, facts.factList.length, 'this is an injected fact');
```

Comment:
This is really minor, and I *love* that you wrote up these tests, but would there be any advantage to casting the injected fact to a variable rather than writing it out each time? I guess I'm thinking if we were to ever want to change the fact, or test against two. It's probably not worth it.

Reply:
Generally speaking, for simple values, I like to use literals in the assertion, at least.  That way your assertion is a little more ironclad.  As much as possible, I like my tests very un-abstract.  Obviously that's no good for more complex tests but 🤷‍♂️.  It probably works out about the same either way.